feconf.py

```diff
@@ -643,3 +643,17 @@ def get_empty_ratings():
 
 CSRF_PAGE_NAME_CREATE_EXPLORATION = 'create_exploration'
 CSRF_PAGE_NAME_I18N = 'i18n'
+
+# pylint: disable=line-too-long
+# The following are all page descriptions for the meta tag.
+ABOUT_PAGE_DESCRIPTION = 'Oppia is an open source learning platform that connects a community of teachers and learners. You can use this site to create 1-1 learning scenarios for others on thare created through modules or explorations.'
```

Comment:
Don't need the line too long thing. Just do:
```
ABOUT_PAGE_DESCRIPTION = (
    'Oppia is an open source learning platform that connects a community of '
    'teachers and learners. You can use this site to create 1-1 learning scenarios '
    'for others on thare created through modules or explorations.')
```

Btw, last part of this sentence doesn't make sense.